commands/addons.js

```diff
@@ -255,36 +254,45 @@ module.exports = {
 
     run:         cli.command(co.wrap(run)),
     usage:       `${topic} [--all|--app APP]`,
-    description: 'lists your add-ons and attachments',
+    description: 'Lists your add-ons and attachments',
```

Comment:
Ah yeah I think this was accidental (file changed on git pull, then my editor re-saved over it). I'll correct it.